build/document-extractor.js

```diff
@@ -460,6 +463,60 @@ function _addSingleSpecialSection($) {
     ];
   }
 
+  /**
+   * @param {object} support
```

Comment:
Could we add an example of what the object will contain? Something like:

```suggestion
   * @param {object} support
   * Example support object:
   * {
   *    version_added: "98.3" || null,
   *    version_removed: "100.0.1" || null
   * }
```

Not sure if there are any other properties in the object as well.

Reply:
If we could use TypeScript there, it would be `bcd.SimpleSupportStatement` instead of `object`. Maybe it would suffice if I added that as a hint?

```suggestion
   * @param {object} support - {bcd.SimpleSupportStatement}
```